src/common/parse.js

```diff
@@ -17,7 +18,7 @@ if (_.isUndefined(parseConfig)) {
 }
 
 Parse.initialize(parseConfig.appId, parseConfig.javascriptKey);
-Parse.CoreManager.set('REQUEST_HEADERS', { 'X-RWALLET-API-KEY': parseConfig.rwalletApiKey, 'X-RWALLET-ENV': parseConfig.rwalletEnv });
+Parse.CoreManager.set('REQUEST_HEADERS', { 'X-RWALLET-API-KEY': parseConfig.rwalletApiKey, 'X-RWALLET-ENV': parseConfig.rwalletEnv, 'X-VERSION': VersionNumber.appVersion });
```

Comment:
1. X-RWALLET-VERSION, you need to create an issue on the server side to remind them of this header.

2. remove 'X-RWALLET-ENV': parseConfig.rwalletEnv